proto/self_serve/orb/v1/orb.proto

```diff
@@ -2,13 +2,26 @@ syntax = "proto3";
 
 package self_serve.orb.v1;
 
+enum OrbMode {
+  ORB_MODE_UNSPECIFIED = 0;
+  LEGACY = 1;
+  SELF_SERVE = 2;
```

Comment:
@andronat do you want me to convert these enums to messages so you can add additional info to each one, eg.:
```
message SelfServe {
   int32 version = 1;
}
```
becase I assume we'll iterate on self serve for a long time before adding another, completely different flow